recipes/cyclus/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set version = "osicbc" %}
+{% set sha256 = "2aa273b4c5bb7be835fdd9132157b3252d60e0543f3ece8f641c5094377d046f" %}
+{% set variant = "openblas" %}
+
+package:
+  name: cyclus
+  version: {{ version }}
+
+source:
+  fn: cyclus-{{ version }}-{{ sha256 }}.tar.gz
+  url: https://github.com/scopatz/cyclus/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  detect_binary_files_with_prefix: true
+  features:
+    - blas_{{ variant }}
+
+requirements:
+  build:
+    - python
+    - cmake
+    - pkg-config
+    - libxmlpp
+    - coincbc
+    - blas 1.1 {{ variant }}
+    - openblas 0.2.18*
+    - boost 1.61.*
+    - hdf5 1.8.17|1.8.17.*
+    - sqlite 3.13.*
+    - pcre
+    - gettext
+    - bzip2 1.0.*
+    - xz 5.2.*
+    - lapack
+  run:
+    - libxmlpp
+    - coincbc
+    - blas 1.1 {{ variant }}
+    - openblas 0.2.18*
+    - boost 1.61.*
+    - hdf5 1.8.17|1.8.17.*
+    - sqlite 3.13.*
+    - pcre
+    - gettext
+    - bzip2 1.0.*
+    - xz 5.2.*
+    - lapack
+
+test:
+  requires:
+    - setuptools
+    - nose
+    - pytables
+about:
+  home: http://fuelcycle.org/
+  license: BSD 3-clause
```

Comment:
Could we please add `license_file: LICENSE.rst` to include the license in the built package? Try to do this so that we are compliant with the BSD license.